src/events/mouse.js

```diff
@@ -929,4 +969,44 @@ p5.prototype._onwheel = function(e) {
   }
 };
 
+/**
+ * <p>The function <a href="#/p5/requestPointerLock">requestPointerLock()</a>
+ * locks the pointer to its current position and makes it invisible.
```

Comment:
Can you add one more sentence here explaining what this function does / why it's useful? Something like "This enables you to..."

Reply:
https://github.com/processing/p5.js/pull/3804/commits/2fcfa818300f93b63b3363efc8670254fe2e0fc6
https://github.com/processing/p5.js/pull/3804/commits/81d7b153e4a978165428688be730d616996c5013  added :

> <p>This enables you to create experiences that aren't limited by the mouse moving out of the screen even if it is repeatedly moved into one direction . </p>
>  <p>For example a first person perspective experience</p>